src/App.js

```diff
@@ -1,10 +1,38 @@
 import React from 'react';
+
 import './App.css';
+import TeamMembers from './components/TeamMembers';
+
+const members = [{
```

Comment:
ja bym przeniósł tych użytkowników do osobnego pliku i tutaj importował - łatwiej czytelniej będzie potem zrobić przepięcie na API

Reply:
Zastanawiam się, czy w zasadzie tego nie usunąć, bo to było tylko do testów, a struktura aplikacji i tak będzie trochę inaczej wyglądać (tzn. TeamMembers będzie w komponencie Content a nie w App).